iceoryx2-ffi/cxx/include/iox2/active_request.hpp

```diff
@@ -139,29 +179,99 @@ template <ServiceType Service,
           typename RequestHeader,
           typename ResponsePayload,
           typename ResponseHeader>
+template <typename T, typename>
 inline auto ActiveRequest<Service, RequestPayload, RequestHeader, ResponsePayload, ResponseHeader>::loan_uninit()
     -> iox::expected<ResponseMutUninit<Service, ResponsePayload, ResponseHeader>, LoanError> {
-    IOX_TODO();
+    ResponseMutUninit<Service, ResponsePayload, ResponseHeader> response;
+    auto result = iox2_active_request_loan_slice_uninit(
+        &m_handle, &response.m_response.m_response, &response.m_response.m_handle, 1);
```

Comment:
What do you think about defining a variable `constexpr uint64_t NUMBER_OF_ELEMENTS = 1;` and using it here. This would provide some context information to other developers.